golden_bridge.rb

```diff
@@ -0,0 +1,61 @@
+class Array
+  def convert_to_integer
+    return [] if empty?
+
+    map(&:to_i)
+  end
+
+  def quicksort
```

Comment:
@Murzic [quicksort](https://en.wikipedia.org/wiki/Quicksort) is an experimental method that I wanted to try instead of usual sort. No big difference for me what to use for sorting because both of them doing the same job. But like a said it's just an experiment.